### ISSUE TITLE
Hosting and Tenant: Multiple domains hosting

### ISSUE THREAD

**Issue by AZholtkevych** [BeaconCMS/beacon#95]
As a PO, I would like to have multiple domain/apps hosting for BeaconCMS and have configuration/options to manage those from the CMS

/Beacon's ability to host multiple domains is a long-term goal, not critical for launch. For now academy.dockyard.com, getfirefly.org, native.live, and beaconcms.org will stay on Netlify/

**Comment by AZholtkevych** [comment]
According to @TheFirstAvenger this may not be issue, since guys taking care about that from CS side
FYI @adam-phillips

**Comment by leandrocp** [comment]
In progress with @alexandrexaviersm 
FYI @AZholtkevych

**Comment by colindensem** [comment]
Following as I have a use case which is for n-tenants.

**Comment by leandrocp** [comment]
@alexandrexaviersm will provide some docs soon. 
TLDR: it works by using the [`:host` option](https://hexdocs.pm/phoenix/Phoenix.Router.html#scope/2)

**Comment by alexandrexaviersm** [comment]
Multiple domains works well on Phoenix, here's a video of it working fine using the app [beacon_demo](https://github.com/BeaconCMS/beacon_demo)

https://www.loom.com/share/016bf19f4b334a2f9312211867c1130c

**Comment by alexandrexaviersm** [comment]
the fork I created: https://github.com/alexandrexaviersm/beacon_demo
